spec/components/schemas.yaml

```diff
@@ -818,6 +800,209 @@ components:
       description: Feature identifier (unique per collection)
       example: smr200-200-4-2019
       type: string
+    itemGeometry:
+      # oneOf:
+        # - $ref: "#/components/schemas/geoJsonPoint"
+        # - $ref: "#/components/schemas/geoJsonLineString"
+      $ref: "#/components/schemas/geoJsonPolygon"
+        # - $ref: "#/components/schemas/geoJsonMultiPoint"
+        # - $ref: "#/components/schemas/geoJsonMultiLineString"
+        # - $ref: "#/components/schemas/geoJsonMultiPolygon"
```

Comment:
I would either remove all comments of at least leave it with the oneOf:
```yaml
oneOf:
  - $ref: "#/components/schemas/geoJsonPolygon"
  #- $ref: "#/components/schemas/geoJsonPoint"
 ...
```

Reply:
If `oneOf` was not commented out here, then we'd have a text in the spec like `one of` + only one choice, which I found did not make too much sense. That's why I commented `oneOf` out until in future probably at least one of the other choices becomes available, too. I don't mind removing all commented out lines here, just left them for convenience, in case they are needed in future.